### PR TITLE
remove confusing statement

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5416

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - We would like to follow up. Can you please do the copy/edit review for this PR? Thanks.

File: windows/deployment/update/waas-servicing-differences.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Windows 10 provided an opportunity to end the era of infinite fragmentation. Wit
 
 This helps simplify servicing. Devices with the original Release to Market (RTM) version of a feature release installed could get up to date by installing the most recent LCU. 
 
-Windows publishes the new LCU packages for each Windows 10 version (1607, 1709, etc.) on the second Tuesday of each month. This package is classified as a required security update and contains contents from the previous LCU as well as new security, non-security and Internet Explorer 11 (IE11) fixes. The security classification, by definition, requires a reboot of the device to complete installation of the update.
+Windows publishes the new LCU packages for each Windows 10 version (1607, 1709, etc.) on the second Tuesday of each month. This package is classified as a required security update and contains contents from the previous LCU as well as new security, non-security and Internet Explorer 11 (IE11) fixes. It could require a reboot of the device to complete installation of the update.
```
Comment (JohanFreelancer9) [inline windows/deployment/update/waas-servicing-differences.md]:
```suggestion
Windows publishes the new LCU packages for each Windows 10 version (1607, 1709, etc.) on the second Tuesday of each month. This package is classified as a required security update and contains contents from the previous LCU as well as new security, non-security and Internet Explorer 11 (IE11) fixes. A reboot of the device might be required to complete installation of the update.
```

**Comment by mypil** [comment]
@MaratMussabekov - please merge the commit on the copy/edit review done by JohanFreelancer9  or make the necessary changes on the PR following the suggestions made. Thank you.

**Pull request merged**